x/interchainstaking/keeper/redemptions.go

```diff
@@ -316,6 +321,9 @@ func DetermineAllocationsForUndelegation(currentAllocations map[string]math.Int,
 			if !ok {
 				availablePerValidator[deltas[idx].ValoperAddress] = sdk.ZeroInt()
 			}
+			fmt.Println("Raw allocation", allocation)
```

Comment:
Add her

Reply:
do we want to remove all fmt.Println here?

Reply:
i can see more